crates/host/src/wasmbus/mod.rs

```diff
@@ -3799,6 +3799,8 @@ impl Host {
 
         let mut links = self.links.write().await;
         links.insert(id.to_string(), ld.clone());
+        // Explicitly drop the lock to avoid deadlock with tasks that lock actors before links
+        drop(links);
```

Comment:
I prefer to wrap in blocks rather than explicitly dropping as it avoids the possibility of accidentally deleting this or moving to the wrong place

Reply:
I think you can also shorthand this by doing:

```rust
self.links.write().await.insert(id.to_string(), ld.clone());
```

That will drop the reference immediately I think?